README.md

```diff
@@ -47,6 +47,7 @@ Following parameters are available to customize the elastic cluster:
   - Using a provisioner
     - type: Defines the type of storage to provision based upon cloud (e.g. `gp2`)
     - storage-class-provisioner: Defines which type of provisioner to use (e.g. `kubernetes.io/aws-ebs`)
+    - encrypted: Whether or not to use encryption. `"true"` or `"false"` Defaults to: `"true"`
```

Comment:
Could we clarify this name a bit? I could see `encrypted` being confusing if it's encryption on storage, or in transit.

Reply:
Sure, I used that name as that is the name k8s itself uses in its storageclass parameters